crates/orchestrator/src/jobs/snos_job/fact_topology.rs

```diff
@@ -73,7 +73,7 @@ pub fn get_page_sizes(pages: &HashMap<usize, PublicMemoryPage>, output_size: usi
         } else {
             ensure!(
                 Some(page_start) == expected_page_start,
-                FactError::OutputPagesUnexpectedStart(page_id, page_start, expected_page_start.unwrap_or_default(),)
+                FactError::OutputPagesUnexpectedStart(page_id, page_start, expected_page_start.unwrap_or_default(),) /* The unwrap here is fine as the assert is exactly for this reason */
```

Comment:
doesn't this always resort to the default value?

Reply:
Yes it will, you want me to change it to `expected_page_start::default()`